components/AddTranslationForm/AddTranslationForm.js

```diff
@@ -0,0 +1,94 @@
+import styled from "styled-components";
+import { InputField, Dropdown, BackButton, ActionButton } from "../StyledForm";
+import { languages } from "../../lib/languages";
+
+export default function AddTranslationForm({
+  onReturnFromEditMode,
+  onSaveTranslation,
+  id,
+}) {
+  function handleAddedTranslation(event) {
+    event.preventDefault();
+    const fields = event.target.elements;
+    const language2 = fields.queryLanguage2.value.split("-");
+    const translation2 = fields.queryLanguage2Translation.value;
+    const gender2 = fields.gender2.value;
+
+    const query2 = {
+      language: language2[1],
+      flag: language2[0],
+      translation: translation2,
+      gender: gender2,
```

Comment:
do you need the 2s in the variable names here? Looks a bit odd - then again, its just the looks...

Reply:
I need to distinguish them from the values of query1... so I would say yes.. but you're right, it looks a bit strange ;)

Reply:
The variables are local to your function (you define them only here), there should be no conflict. 

Of course the name prop after "fields." must be the same as in your jsx